### PR TITLE
specify openhab2 package version

### PR DESCRIPTION
**Pull request by mstormi**
required if moving from snapshot to milestone else won't obtain that.

Signed-off-by: Markus Storm markus.storm@gmx.net (github: mstormi)

### CODE REVIEW

**Comment by mstormi** [comment]
you mean the -y ? It's been in there for as long as I know.
Large or not, it's only happening after an active user choice plus another safety check question, so I feel no, we shouldn't ask again.

**Comment by ThomDietrich** [comment]
@BClark09 wdyt?

File: functions/openhab.sh [hunk 1]
```diff
@@ -50,7 +50,8 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
 
   echo "$REPO" > /etc/apt/sources.list.d/openhab2.list
   cond_redirect apt update
-  cond_redirect apt -y install openhab2
+  local openhabVersion="$(apt-cache madison openhab | head -n 1 | cut -d'|' -f2 | xargs)"
```
Comment (EliasGabrielsson) [inline functions/openhab.sh]: Change to `apt-cache madison openhab2 | head -n 1 | cut -d'|' -f2 | xargs`

**Comment by EliasGabrielsson** [comment]
@mstormi 
When indicating that something shall work make sure to **actually test** the code yourself before submitting your PRs onwards. It's okay to submit unfinished code but you need to mark it (WIP or similar) so we can put or spare voluntary time where it make best use.

**Comment by EliasGabrielsson** [comment]
I think we can keep the "downgrade" risk mention by Clark. It's a know pattern and people should be aware that going from test-build to stable is not a supported usecase.

**Comment by BClark09** [comment]
I agree with what's been said, happy to merge after `openhab` is changed to `openhab2` (a typo propagated from our discussion in #475

**Comment by mstormi** [comment]
sorry for that one slipping me

**Pull request merged**